### DISCUSSION TITLE
Should --get throw a warning if a variable is unknown

### DISCUSSION THREAD

**Discussion by jacobmealey** [curl/trurl#236] [Ideas]
currently if you call `trurl --url "https://example/com" -g "{host} {foo}"` it will print `example `. I think it should either print `example {foo}` or `example ` to stdout and some type of warning to stderr that says "Unkown variable foo". I'm leaning towards the warning to stderr and using `trurl_warnf` like in #231.

**Comment by bagder** [comment]
A third option is to error out completely, to make it more apparent if you spelled a keyword wrong etc...

I have a feeling we are not going to add many more keywords going forward so there is little risk that you're going to end up in a situation where you run an older version using a newer command line and thus using a keyword not yet implemented.

**Reply by jacobmealey** [reply]
Okay, that makes sense. 👍

**Comment by bagder** [comment]
See  #286
